tests/unit/query_cost_estimator.cpp

```diff
@@ -201,7 +201,7 @@ TEST_F(QueryCostEstimator, SubqueryCartesian) {
   std::shared_ptr<LogicalOperator> input = std::make_shared<ScanAll>(std::make_shared<Once>(), NextSymbol());
   std::shared_ptr<LogicalOperator> subquery = std::make_shared<ScanAll>(std::make_shared<Once>(), NextSymbol());
   MakeOp<memgraph::query::plan::Apply>(input, subquery, true);
-  EXPECT_COST(CostParam::kSubquery * no_vertices * no_vertices);
+  EXPECT_COST(CostParam::kSubquery * no_vertices * no_vertices + no_vertices);
```

Comment:
Why the subquery cost changed?

Reply:
It was wrong, I made this test a while ago, but I think it's better like this

consider MATCH (m) CALL { MATCH (n) RETURN n } RETURN n, m
if there are 4 nodes, it should be 4 results after match + 16 results at the end of the query, therefore the cardinality should be 16 but cost is 4 + 16